src/events/nats.js

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable camelcase */
+/**
+ * @fileoverview Instrumentation for nats library.
+ */
+const shimmer = require('shimmer');
+const tracer = require('../tracer.js');
+const eventInterface = require('../event.js');
+const moduleUtils = require('./module_utils.js');
+
+const NATS_TYPES = {
+    name: 'nats',
+    mainWrappedFunction: 'Client',
+    badMessage: 'NATS_BAD_JSON_MSG',
+};
+
+const getServerHostname = (currentServer) => {
+    let serverHostname = NATS_TYPES.serverDefaultHostname;
```

Comment:
you can use the trinary operator ( `a ? b : c`) here instead

Reply:
I think its more readable and clearer instead: 
`const getServerHostname = (currentServer) => (currentServer.url && currentServer.url.hostname) ? currentServer.url.hostname : NATS_TYPES.serverDefaultHostname
`

Reply:
changed.